<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
        backgound-image; url(:/poazdina/pozadina.png);
        background-repat no-repat;
       backgound-position: center;


}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SMART GARDEN</string>
   </property>
  </widget>
  <widget class="QLabel" name="labela_vreme">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labela_lm35">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 201, 255);
color: rgb(60, 118, 255);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pomoc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="pomoc.qrc">
     <normaloff>:/pomoc/pomoc.png</normaloff>:/pomoc/pomoc.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>78</width>
     <height>308</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="termostat">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="termostat.qrc">
        <normaloff>:/termostat/termostat.png</normaloff>:/termostat/termostat.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sijalica">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="sijalica.qrc">
        <normaloff>:/sijalica/sijalica.png</normaloff>:/sijalica/sijalica.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ventilator">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="ventilator.qrc">
        <normaloff>:/ventilator/ventilator.png</normaloff>:/ventilator/ventilator.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="zalivanje">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zalivanje.qrc">
        <normaloff>:/zalivanje/zalivanje.png</normaloff>:/zalivanje/zalivanje.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="vlaznostZemljista">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>118</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_ldr">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 248, 219);
color: rgb(230, 153, 0);
border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="manuelno">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Manuelno</string>
   </property>
  </widget>
  <widget class="QPushButton" name="automatski">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Automatski</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="zalivanje.qrc"/>
  <include location="sijalica.qrc"/>
  <include location="ventilator.qrc"/>
  <include location="pomoc.qrc"/>
  <include location="termostat.qrc"/>
 </resources>
 <connections/>
</ui>
